integration.md

```diff
@@ -81,30 +81,32 @@ We've previously stated that the forms presented by Klarna are eventually posted
 ```ruby
 # Handle POST requests to '/purchase'
 post '/purchase' do
+  @user = User.create(session, params[:userToken], 'user@email.com') unless @user
   purchase = Klarna::Purchase.new(
-    user_token:    params[:userToken],
-    reference:     "IA-#{params[:article_id]}",
-    name:          'Interesting Article',
+    user_token:    @user.token,
+    reference:     'subscription',
```

Comment:
It was easier to create another pull request:
https://github.com/klarna/on-demand-for-digital-goods-docs/pull/3/files

Just note that the code on the doc should be simpler than the code on the site, which really needs to work :) and have site logic(users, sessions, ...)